service/github/github.go

```diff
@@ -321,18 +321,39 @@ func (g *PullRequest) diffUsingGitCommand(ctx context.Context) ([]byte, error) {
 		return nil, err
 	}
 
-	headSha := pr.GetHead().GetSHA()
+	head := pr.GetHead()
+	headSha := head.GetSHA()
 
-	mergeBaseBytes, err := exec.Command("git", "merge-base", headSha, pr.GetBase().GetSHA()).Output()
+	commitsComparison, _, err := g.cli.Repositories.CompareCommits(ctx, g.owner, g.repo, headSha, pr.GetBase().GetSHA(), nil)
 	if err != nil {
-		return nil, fmt.Errorf("failed to get merge-base commit: %w", err)
+		return nil, err
 	}
 
-	mergeBase := strings.Trim(string(mergeBaseBytes), "\n")
+	mergeBaseSha := commitsComparison.GetMergeBaseCommit().GetSHA()
+
+	if os.Getenv("REVIEWDOG_SKIP_GIT_FETCH") != "true" {
+		remoteRepo := "reviewdog_origin"
```

Comment:
Instead of adding remote repo, cannot we use `git fetch --depth=1 head.GetRepo().GetHTMLURL() {SHA}`?

Reply:
I do not know it.
I apply it: https://github.com/reviewdog/reviewdog/pull/1714/commits/fdddf8fd18043371d388cf93ec7a0b6463942706